### ISSUE TITLE
Font size sign-up page helptext large enough.

### ISSUE THREAD

**Issue by brucestull** [brucestull/technology-and-perceptibility#106]
The font size on sign-up page should be large enough for accessibility and scaling with user settings.

Possible CSS:
```
span.helptext {
  font-size: 1.25em;
}
```

Reference:
`<span class="helptext">Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.</span>`
`<span class="helptext">Enter the same password as before, for verification.</span>`
